### PR TITLE
Adds a list of the events each section describes and reformats the context member events

### PR DESCRIPTION
**Pull request by lamagnifica**
@mhoeber @catong @srpearce I added some organizational help to this chapter. 
- Reformatted the context member fields into a table
- added a list of the events that each section describes
- added some refs back to the common context field
- fixed a few typos
  Just looking for a check that I didn't delete anything inadvertently or otherwise mess up...

### CODE REVIEW

**Comment by srpearce** [comment]
@mhoeber @lamagnifica @catong I can get this one.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 1]
```diff
@@ -10,28 +10,30 @@ the mobile device to capture information about interactions with the courseware
...
-  in the LMS.
-
-Student and instructor events are grouped into categories in this chapter. For
-a list of events, see the :ref:`event_list`.
+* A :ref:`sample event<sample_events>`.
+* The :ref:`common fields<common>` that are included in the JSON document of
+  every event.
+* Descriptions of :ref:`student initiated events<Student_Event_Types>`, which
+  are generated by interactions with the learning management system (LMS)
+  other than the Instructor Dashboard.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: nit: extra space here?

Also, this wording seems kind of awkward. Maybe:

Descriptions of student-initiated events. Interactions with all pages in the Learning Management System (LMS) except for the Instructor Dashboard generate these events.

or

Descriptions of student-initiated events, which are generated by all interactions with the Learning Management System (LMS) except for interactions with the Instructor Dashboard.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 2]
```diff
@@ -10,28 +10,30 @@ the mobile device to capture information about interactions with the courseware
...
+* Descriptions of :ref:`course team initiated events<Instructor_Event_Types>`,
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: course team-initiated

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 3]
```diff
@@ -10,28 +10,30 @@ the mobile device to capture information about interactions with the courseware
...
+  which are generated by interactions with the Instructor Dashboard in the
+  LMS.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Descriptions of course team-initiated events. Interactions with the Instructor Dashboard in the LMS generate these events. (note hyphen in first sentence)

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 4]
```diff
@@ -184,39 +194,82 @@ more information, see the HTTP/1.1 header field definition for
...
-For all events, this field includes member fields that provide the following
-contextual information.
+The ``context`` field includes member fields that provide contextual
+information. 
+
+* This field contains a core set of member fields that are common to all events.
+* For certain events with additional contextual requirements, this field
+  contains a set of additional member fields that are common to those events
+  only.
+* For any event, this field can also include one or more additional member
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:209]: Need another line break here?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 5]
```diff
@@ -184,39 +194,82 @@ more information, see the HTTP/1.1 header field definition for
...
+       ``usage_key``.
+
+       For modules that are used in a course to present content from a
+       library, this dictionary also includes the ``original_usage_key`` and
+       ``original_usage_version`` fields. These member fields provide a
+       consistent way to identify components that are sourced from a library,
+       and can be used to identify the source library.
+
+
+The ``context``  member fields are blank if values cannot be determined.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Extra space before "member"?

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 6]
```diff
@@ -611,34 +697,35 @@ All of the navigational events add the same fields to the ``event`` dict field.
...
+
+A browser or the edX mobile app emits video interaction events when a user
+interacts with a video.
 
 * When users use a browser to stream video files on a desktop computer or
   mobile device, the browser emits the events.
 
 * When users use the edX mobile app to stream or download course videos for
-  offline viewing, the mobile app emits the events.
+  off line viewing, the mobile app emits the events.
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Any reason for this change? I've always seen it as "offline" (as opposed to "online"), and wikipedia has the following:

The terms "online" and "offline" have specific meanings in regard to computer technology and telecommunications in which... "offline" indicates a disconnected state.

File: en_us/data/source/internal_data_formats/tracking_logs.rst [hunk 7]
```diff
@@ -611,34 +697,35 @@ All of the navigational events add the same fields to the ``event`` dict field.
...
 
-  When a user interacts with a downloaded video file offline using the edX
+  When a user interacts with a downloaded video file off line using the edX
```
Comment (srpearce) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: Same question as above.

**Comment by srpearce** [comment]
A few nits - otherwise :+1:

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 3 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: I was trying to follow the decision in our last style guide meeting, to minimize the use of hyphens. See http://draft-edx-style-guide.readthedocs.org/en/latest/preferred.html. Reworded instead.

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 6 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: yep. note that we will all have to "add" these spellings, as the spell checker in Sublime picks them up as incorrect

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 1 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst]: extra space is not visible in Sublime. I tried to get rid of it, but won't spend any more time chasing it in future commits.

File: en_us/data/source/internal_data_formats/tracking_logs.rst (hunk 4 above)
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/tracking_logs.rst:209]: I'm not seeing any problem with the spacing. What is the purpose of the new line between 208 and 209?

**Pull request merged**